ldes-server-port-fetch-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/rest/treenode/services/EventStreamInfoResponse.java

```diff
@@ -51,4 +53,12 @@ private boolean hasMeaningfulValue(String objectContent) {
 		return objectContent != null && !objectContent.equals("");
```

Comment:
I know this line is not written by you, but while reviewing this PR, my eye caught this and I was wondering if you could replace this with `.isEmpty()`